minitwit/src/main/java/WebApplication.java

```diff
@@ -387,6 +396,116 @@ public static void updateLatest(Request request) {
         return "";
     };
 
+    public static Route serveSimFllws = (Request request, Response response) -> {
+
+        //update_latest(request); //TODO: call
+
+        var db = new SQLite();
+        var conn = db.getConnection();
+
+        var who_id = getUserID(db, request.params(":username"));
+
+        if (who_id == 0) {
+            conn.close();
+            response.status(404);
+            return false;
+        }
+
+        Gson gson = new Gson();
+        Fllws fllws = gson.fromJson(request.body(), Fllws.class);
+
+
+        if (Objects.equals(request.requestMethod(), "POST") && !fllws.follow.isEmpty()) {
+
+            var whom_id = getUserID(db, fllws.follow);
+
+            if (whom_id == 0) {
+                conn.close();
+                response.status(404);
+                return false;
+            }
+
+
+            var insert = conn.prepareStatement("insert into follower (\n" +
```

Comment:
Is `\n` valid SQL? It's not in the reference API.

Reply:
Its how we had done the original SQL queries, and since the API and follow/unfollow page used the same queries, i just copied them over from there. It is a bit weird, so i have changed it to a single line.

Reply:
Ah I see, probably fine then.